.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish Release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Java 11 and Apache Maven
+        uses: actions/setup-java@v3
+        with:
+          java-version: 11
```

Comment:
Please add `distribution`. For more information: https://github.com/actions/setup-java/blob/main/docs/switching-to-v2.md#java-distribution

Reply:
done